mappings/net/minecraft/menu/GenericMenu.mapping

```diff
@@ -1,8 +1,20 @@
 CLASS net/minecraft/class_1707 net/minecraft/menu/GenericMenu
+	COMMENT A generic 9xN menu with a customizable amount of rows.
+	COMMENT
+	COMMENT <p>The {@code createGeneric9xN} methods should usually be used only on the client,
+	COMMENT as they have an empty inventory by default.
```

Comment:
Maybe mention that they can be updated by s2c packets and have c2s packets to response?

Reply:
I think that that would work better in the doc on `Menu`.

Reply:
I think the comment I have on `Menu` already describes the packet behavior (well, maybe not c2s responses but that's an impl detail).

Reply:
worth nothing that server-only fab mods can send a simple generic menu to client and the client will have no trouble interacting with the menu